wiki/People/osu!_team/en.md

```diff
@@ -42,33 +52,36 @@ In addition to the above, the [account support team](/wiki/People/Account_suppor
 
 | Name | Role |
 | :-- | :-- |
-| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | General housekeeper and helper, chibi artist. [Barrel Roll Weekly](http://brw.twinkfish.com/) |
-| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | Moderation and community advisor |
-| ::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) | Audio designer, [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| ::{ flag=CA }:: [awp](https://osu.ppy.sh/users/2650) | General housekeeper and helper, chibi artist |
+| ::{ flag=US }:: [Chaos](https://osu.ppy.sh/users/2628870) | Moderator |
+| ::{ flag=NO }:: [cYsmix](https://osu.ppy.sh/users/272870) | Audio designer, [Featured Artist](https://osu.ppy.sh/beatmaps/artists/2) |
+| ::{ flag=US }:: [Daru](https://osu.ppy.sh/users/32480) | osu!painter |
 | ::{ flag=NZ }:: [deadbeat](https://osu.ppy.sh/users/128370) | World Cup organiser and administrator |
 | ::{ flag=US }:: [Derekku](https://osu.ppy.sh/users/91341) | General housekeeper, community manager |
-| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | osu! developer, provider of IRC integration for in-game chat, site maintainer. [Blog](http://blog.echo.sh/) |
-| ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour host, World Cup streamer, tournament organizer |
+| ::{ flag=FR }:: [Damnae](https://osu.ppy.sh/users/989377) | Storyboard developer |
+| ::{ flag=NZ }:: [Echo](https://osu.ppy.sh/users/431) | osu! developer, provider of IRC integration for in-game chat, site maintainer |
+| ::{ flag=US }:: [HappyStick](https://osu.ppy.sh/users/256802) | osu! Coffee Hour host, World Cup streamer, tournament organiser |
 | ::{ flag=NL }:: [Intermezzo](https://osu.ppy.sh/users/136842) | osu! developer, provider of osz2 and p2p backend |
-| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) | Modding and mapping scene advisor |
-| ::{ flag=US }:: [Datawalke](https://osu.ppy.sh/users/142) | Original site designer, hosting provider in the early days. [Brave New Games](http://www.bravegamer.com/) |
+| ::{ flag=GB }:: [JBHyperion](https://osu.ppy.sh/users/4879508) | Community leader |
```

Comment:
Adding the community they led/lead is probably important so its not under the assumption they are like Ephemeral.

Reply:
fixed

Reply:
> Adding the community they led/lead is probably important so its not under the assumption they are like Ephemeral.

I +1 this one, anyone not familiar with JBHyperion will assume their job is the same as Eph's